src/components/Bookinput.jsx

```diff
@@ -20,21 +20,20 @@ function Bookinput() {
     const id = uuidv4();
     e.preventDefault();
     if (title.trim() && author.trim()) {
+      const book = { id, title, author };
       dispatch(
-        addnewBook(id, title, author),
+        addBook(book),
       );
     } else {
-      dispatch(
-        addnewBook(id, title, author),
-      );
+      return;
     }
     setTitle('');
     setAuthor('');
   };
   return (
     <form action="#" onSubmit={Submit}>
-      <input type="text" placeholder="book title" onChange={handletitle} />
-      <input type="text" placeholder="auther" onChange={handleauthor} />
+      <input type="text" value={title} placeholder="book title" onChange={handletitle} />
+      <input type="text" value={author} placeholder="auther" onChange={handleauthor} />
       <button type="submit">submit</button>
     </form>
   );
```

Comment:
[Screencast from 08-26-2022 03:19:51 AM.webm](https://user-images.githubusercontent.com/65068771/186798035-eef08171-3d15-45a7-8df9-392cac53f574.webm)

- [ ] Kindly fix these errors in the screencast video above that occur when a new book is created and deleted.